server/routes/projects/read.js

```diff
@@ -44,12 +44,14 @@ module.exports = function(config, req, res, next) {
       }
 
       if (response.statusCode === 404) {
-        // If there aren't any projects for this user, create one with a redirect
-        res.redirect(307, "/" + locale + "/projects/new" + qs);
-        return;
-      }
-
-      if (response.statusCode !== 200) {
+        if (config.shutdownNewProjectsAndPublishing) {
+          body = "[]";
```

Comment:
it feels like it might be more useful to have this be a redirect to the "we've moved to Glitch" page rather than simply presenting an empty list?